src/content/api/agent/v1.1/index.apib

```diff
@@ -97,6 +97,34 @@ The agent is asked to perform any necessary operations before the broker execute
 + Response 500
 <!-- include(responses/500.apib) -->
 
+# Group PreUpdate
+Whenever someone asks the broker to update a service instance, it commissions an agent to perform any operations required to be executed before the actual update of the instance gets triggered.
+
+## PreUpdate [POST /preupdate]
+Called when ```cf update-service``` is invoked on a BOSH service instance.
+The agent is asked to perform any necessary operations before the broker executes the update of the service instance.
+
++ Parameters
+    + BODY.deployment_name: `` (string) - Deployment Name
+    + BODY.instance_guid: `` (string) - Instance Guid of service instance
+    + BODY.params: `` (object) - params that comes as part of cf update request, includes context, plan_id, service_id, previous_values and previous_manifest
```

Comment:
Need more elaboration here. I would be better if you expand the schema of this params object here like:
```
BODY.params.context - OSBAPI spec context object information
BODY.params.plan_id - 
BODY.params. service_id - 
BODY.params. previous_values - 
BODY.params.previous_manifest - 
Additionally any command line json args as passed in the `cf update-service` operation can be found under BODY.params
```

Reply:
Agree, missed that part. I'll add